statsd-exporter/include/mappings-gen2.yml

```diff
@@ -101,6 +101,14 @@ mappings:
     name: "airflow_executor_queued_tasks"
     labels: {}
 
+  - match: airflow.executor.*.resources.*
+    name: airflow_runner_resources
+    labels: {}
```

Comment:
If labels are left empty like this then there will just be a single metric in chronosphere named `airflow_runner_resources` with a mix of different `target_name`s and `resource`s.

Shouldn't we have labels like:

```
target_name: "$1"
resource: "$2"
```

to extract those labels from the named labels?

Reply:
Ah, ok. That's what the labels are more! Makes sense. Will push an update.